app/ts/background/windows/personalSign.ts

```diff
@@ -112,54 +113,86 @@ export async function craftPersonalSignPopupMessage(ethereumClientService: Ether
 	}
 	const namedParams = { param: originalParams.params[1], account: originalParams.params[0] }
 	const account = getAddressMetaData(namedParams.account, userAddressBook)
-
-	if (namedParams.param.primaryType === 'Permit') {
-		const parsed = EIP2612Message.parse(namedParams.param)
-		const token = await getTokenMetadata(ethereumClientService, parsed.domain.verifyingContract)
-		const owner = getAddressMetaData(parsed.message.owner, userAddressBook)
-		if (token.type === 'NFT') throw 'Attempted to perform Permit to an NFT'
+	
```

Comment:
Nitpick.  I thought our .editorconfig dealt with this automatically?
```suggestion

```

Reply:
hmm I think it should